src/includes/enriching-events/set-user/android.mdx

```diff
@@ -19,6 +19,6 @@ Sentry.setUser(user)
 
 <Note>
 
-Sentry will by fallback to a random generated `installationId` if you do not provide a user identity.
+If you do not provide a user identity, the SDK falls back to `installationId`, which the SDK once randomly generates during an app's installation.
```

Comment:
any reason to change the wording, previous text has been reviewed by the docs team already and seems clear to me

Reply:
I thought that maybe some users won't understand that the `installationId` stays the same. I can also revert it back to the old version.

Reply:
no strong feeling either, maybe @imatwawana find it even better, just wanna understand the motivation, but I recall being more specific about that, and Fiona reworded this way

Reply:
Given that @philipphofmann wants to convey more information, I think the second is fine with a few tweaks. And the previous sentence didn't entirely make sense anyway. The word "by" shouldn't be there.

```suggestion
If you do not provide a user identity, the SDK falls back to `installationId`, which the SDK randomly generates once during an app's installation.
```